sci-libs/geographiclib/geographiclib-1.49.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit versionator
+inherit cmake-utils
+
+PYTHON_COMPAT=( python{2_7,3_4} )
```

Comment:
is `3_5` possible as well?

Reply:
Sure, there should be no reason not to, other than I haven't really tested it. And also that this package installs the python scripts into a non-versioned `site-packages/python` path.